src/generateData.js

```diff
@@ -6,6 +6,16 @@ const ANIMATION_COUNT = danceParty.constants.MOVE_NAMES.length;
 const FRAMES_PER_ANIMATION = danceParty.constants.FRAMES;
 const SPRITE_NAMES = danceParty.constants.SPRITE_NAMES;
 
+// inferred from /dashboard/config/blocks/Dancelab/Dancelab_setBackgroundEffect.json
+const BG_EFFECT_NAMES = [
```

Comment:
Should these be exported from https://github.com/code-dot-org/dance-party/blob/master/src/constants.js?

Reply:
Probably; given that this is just a QOL data gen script, I'm not considering that a high priority right now